pyvista/plotting/helpers.py

```diff
@@ -462,3 +462,52 @@ def plot_itk(mesh, color=None, scalars=None, opacity=1.0, smooth_shading=False):
     else:
         pl.add_mesh(mesh, color, scalars, opacity, smooth_shading)
     return pl.show()
+
+
+def view_vectors(view: str, negative: bool = False) -> Tuple[np.ndarray, np.ndarray]:
+    """Given a plane to view, return vectors for setting up camera.
+
+    Parameters
+    ----------
+    view : {'xy', 'yx', 'xz', 'zx', 'yz', 'zy'}
+        Plane to return vectors for.
+
+    negative : bool
+        Whether to view from opposite direction.  Default ``False``.
+
+    Returns
+    -------
+    direction : numpy.ndarray
+        [x, y, z] vector that points in the viewing direction
+
+    viewup : numpy.ndarray
+        [x, y, z] vector that points to the viewup direction
+
+    """
+    if view == 'xy':
+        vec = np.array([0, 0, 1])
+        viewup = np.array([0, 1, 0])
+    elif view == 'yx':
+        vec = np.array([0, 0, -1])
+        viewup = np.array([1, 0, 0])
+    elif view == 'xz':
+        vec = np.array([0, -1, 0])
+        viewup = np.array([0, 0, 1])
+    elif view == 'zx':
+        vec = np.array([0, 1, 0])
+        viewup = np.array([1, 0, 0])
+    elif view == 'yz':
+        vec = np.array([1, 0, 0])
+        viewup = np.array([0, 0, 1])
+    elif view == 'zy':
+        vec = np.array([-1, 0, 0])
+        viewup = np.array([0, 1, 0])
+    else:
+        raise ValueError(
```

Comment:
Codecov claims this error needs a test case.

Reply:
Good catch.  We also lacked specific tests for this function.  Added tests.